manager/integration/tests/test_orphan.py

```diff
@@ -40,7 +41,8 @@ def crate_disks_on_host(client, disk_names, request):  # NOQA
         update_disks[name] = disk
         disk_paths.append(disk_path)
 
-    node = node.diskUpdate(disks=update_disks)
+    node = update_node_disks(client, node.name, disks=update_disks,
```

Comment:
I found the enclosed function name is typo :slightly_smiling_face: 

`crate_disks_on_host(` should be `create_disks_on_host(`